fix: cross-platform tracing (#1068)
n1ru4l/envelop: .changeset/spotty-laws-wash.md

---
'@envelop/core': patch
---

Prefer `globalThis.performance.now` for tracing if available. Fallback to `Date.now`.

> Using tracing now no longer raises a errors on browser, deno and cloudflare workers.

Author: ```suggestion
Using tracing no longer raises an error on browser, deno and cloudflare worker environments.
```
